app/models/case_log.rb

```diff
@@ -431,6 +432,15 @@ def get_totadult
     end
   end
 
+  def get_refused
+    return 1 if [age1_known, age2_known, age3_known, age4_known, age5_known, age6_known, age7_known, age8_known].any?(1)
+    return 1 if [sex1, sex2, sex3, sex4, sex5, sex6, sex7, sex8].any?("R")
+    return 1 if [relat2, relat3, relat4, relat5, relat6, relat7, relat8].any?(3)
+    return 1 if [ecstat1, ecstat2, ecstat3, ecstat4, ecstat5, ecstat6, ecstat7, ecstat8].any?(10)
```

Comment:
Would it make sense to have methods for these and do something like:
`return 1 if age_refused? || gender_refused? ...`

Reply:
Might just end up being a one liner instead of a method 👀

Reply:
Updated, good shout - knew I forgot to do something 😄